ghost/members-csv/lib/unparse.js

```diff
@@ -14,53 +14,53 @@ const DEFAULT_COLUMNS = [
     'tiers'
 ];
 
-const unparse = (members, columns = DEFAULT_COLUMNS.slice()) => {
+const unparse = (rows, columns = DEFAULT_COLUMNS.slice()) => {
     columns = columns.map((column) => {
         if (column === 'subscribed') {
             return 'subscribed_to_emails';
         }
         return column;
     });
-    const mappedMembers = members.map((member) => {
-        if (member.error && !columns.includes('error')) {
+    const mappedRows = rows.map((row) => {
+        if (row.error && !columns.includes('error')) {
             columns.push('error');
         }
 
         let labels = '';
-        if (typeof member.labels === 'string') {
-            labels = member.labels;
-        } else if (Array.isArray(member.labels)) {
-            labels = member.labels.map((l) => {
+        if (typeof row.labels === 'string') {
+            labels = row.labels;
+        } else if (Array.isArray(row.labels)) {
+            labels = row.labels.map((l) => {
                 return typeof l === 'string' ? l : l.name;
             }).join(',');
         }
 
         let tiers = '';
 
-        if (Array.isArray(member.tiers)) {
-            tiers = member.tiers.map((tier) => {
+        if (Array.isArray(row.tiers)) {
+            tiers = row.tiers.map((tier) => {
                 return tier.name;
             }).join(',');
         }
 
         return {
-            id: member.id,
-            email: member.email,
-            name: member.name,
-            note: member.note,
-            subscribed_to_emails: member.subscribed || member.subscribed_to_emails ? true : false,
-            complimentary_plan: member.comped || member.complimentary_plan,
-            stripe_customer_id: _.get(member, 'subscriptions[0].customer.id') || member.stripe_customer_id,
-            created_at: member.created_at,
-            deleted_at: member.deleted_at,
+            id: row.id,
+            email: row.email,
+            name: row.name,
+            note: row.note,
+            subscribed_to_emails: 'subscribed' in row ? row.subscribed : row.subscribed_to_emails,
```

Comment:
I'm 99% sure there's no way currently to get a value in row.subscribed, BUT on the off chance I'm missing something here, I changed this up to more explicitly check if the key exists in the row (even if the value is undefined) vs using the `||` operator.

This allows the expected value (`row.subscribed_to_emails`) to be undefined instead of coercing it to a boolean.